### ISSUE TITLE
`require-presentational-children` rule does not like SVG

### ISSUE THREAD

**Issue by mydea** [ember-template-lint/ember-template-lint#2136]
I have an inline SVG like this in my .hbs file:

```hbs
<svg
  width='194'
  height='191'
  viewBox='0 0 194 191'
  xmlns='http://www.w3.org/2000/svg'
  role='img'
>
  <title>
    No report generated yet
  </title>
  <circle cx='95' cy='95' r='95' fill='#F5F7FA'></circle>
  {{! more stuff here... }}
</svg>
```

This triggers these linting errors:

```
  4:6  error  <svg> has a role of img, it cannot have semantic descendants like <title>  require-presentational-children
  4:6  error  <svg> has a role of img, it cannot have semantic descendants like <circle>  require-presentational-children
```

As far as I see, this _should_ be OK, and should be the preferred way to mark up an image-like SVG. I guess there is probably some special handling required for SVG - I'll look into it if I can figure it out!

**Comment by bmish** [comment]
CC: @faith-or
